source/includes/fundamentals/code-examples/linq.cs

```diff
@@ -0,0 +1,38 @@
+
+// start-restaurant-model
+public class Restaurant
+{
+    public ObjectId Id { get; set; }
+
+    public string Name { get; set; }
+
+    [BsonElement("restaurant_id")]
+    public string RestaurantId { get; set; }
+
+    public string Cuisine { get; set; }
+
+    public Address Address { get; set; }
+
+    public string Borough { get; set; }
+
+    public List<Grade> Grades { get; set; }
```

Comment:
Where is the `Grade` class declared?

Reply:
Changed to GradeEntry (due to there being a field called Grade in it) and added the model to the page